StrongInject.Tests.Unit/StrongInject.Tests.Unit.csproj

```diff
@@ -5,6 +5,7 @@
 		<LangVersion>preview</LangVersion>
 		<Nullable>enable</Nullable>
 		<Configurations>Debug;Release;Wpf;All</Configurations>
+    <RootNamespace>StrongInject.Generator.Tests.Unit</RootNamespace>
```

Comment:
I actually cannot use VS to create this line. It forces tabs to be converted to spaces.

Reply:
An .editorconfig file could possibly fix this if you absolutely must use tabs. :P

Reply:
Correction: you can't even do this in another editor so long as VS is _open_.

Reply:
Feel free to convert to spaces in all csprojs if that helps.

Reply:
:+1: That's my preference, and that seems to match all the source files.